src/serum-tokenomics.yaml

```diff
@@ -1,19 +1,349 @@
+      - Holders of unlocked tokens voluntarily moved tokens into reserves, reducing circulating supply, with plans to re-introduce tokens into circulating only to provide liquidity to extraordinary buy-side demand
+      - 825m was set aside for ecosystem incentives for SRM holders and Serum users
+      - Currently, the circulating supply is approximately 125m
+
+
+    stepItems: []
+  - uuid: 2b0067ea-83a5-49cf-a00c-adda5f33957b
+    name: Evaluation
+    content: |
+    stepItems:
+      - uuid: 188364d9-da34-494c-bb4f-8c1100ecea3e
+        content: What is the total supply of Serum (SRM)?
+        answerKeys:
+          - B
+        type: SingleChoice
+        choices:
+          - content: 1,000,000,000
+            key: A
+          - content: 10,000,000,000
+            key: B
+          - content: 100,000,000,000
+            key: C
+          - content: 100,000,000
+            key: D
+
+      - uuid: 938d0ad8-eeaf-41b5-b022-7faaf51e2982
+        content: What was the initial circulating supply of Serum (SRM)?
+        answerKeys:
+          - C
+        type: SingleChoice
+        choices:
+          - content: 125m tokens
+            key: A
+          - content: 825m tokens
+            key: B
+          - content: 175m tokens
+            key: C
+          - content: 100m tokens
+            key: D
+
+  - uuid: dc8b0b36-5003-49e9-a69a-260ceafd1deb
+    name: Fees
+    content: |
+      When placing trades that use Serum's order book, you may be charged fees by the application. Other applications, such as Raydium and Bonfida, may also charge their own fees in addition to any fees Serum charges. All fees collected go to the benefit of SRM. 100% of the exchange fees flow back into SRM, generally via a buy and burn, and also potentially into staking rewards or other ecosystem grants. Node staking has governance over the use of fees in Serum and nodes may alter rules around the usage at any time.
+
+      The fees on trades are (taker/maker):
+      - Hold < 100 SRM: 4 bps / 0 bps
+      - 100 SRM: 3.9 bps / 0 bps
+      - 1,000 SRM: 3.8 bps / 0 bps
+      - 10,000 SRM: 3.6 bps / 0 bps
+      - 100,000 SRM : 3.4 bps / 0 bps
+      - 1m SRM: 3.2 bps / 0 bps
+      - 1 MSRM: 3 bps / 0 bps
+
+      Utilizing Generated Fees:
+      - 68% of net fees go to buy and burn of SRM
+      - 20% of net fees to the project or GUI hosting activity
+      - 10% go to nodes as staking rewards
+      - 2% goes to Ecoserum
+
+
+    stepItems: []
+  - uuid: fe253056-b1fa-48af-bede-51b1e9fa66c5
+    name: Evaluation
+    content: |
+    stepItems: 
+      - uuid: 6863739d-aeb7-4e92-8245-3ec9b4c1b81c
+        content: What is the maker fees on any amount of SRM trade?
+        answerKeys:
+          - C
+        type: SingleChoice
+        choices:
+          - content: 4 bps
+            key: A
+          - content: 3 bps
+            key: B
+          - content: 0 bps
```

Comment:
Is the fee `0` in all scenarios?

Reply:
I used this as a reference for fees: https://docs.projectserum.com/appendix/fees.
Here maker fees is mentioned as 0 in every case